lib/utils/config/definitions.js

```diff
@@ -278,7 +279,7 @@ define('browser', {
   defaultDescription: `
     OS X: \`"open"\`, Windows: \`"start"\`, Others: \`"xdg-open"\`
   `,
-  type: [null, Boolean, String],
+  type: BooleanOrString,
```

Comment:
It can still be null right?  That's how the default works?

Reply:
meant to mark this as draft as i was still experimenting with some things, but what I've found so far:

- `default: null` doesn't require that `type` contains `null`
- if `type` contains `null` and not `Boolean` then any `false` value is coerced back to null. so `--no-browser` would end up `null`, which wouldn't break anything currently but i think is a major footgun

Reply:
what if a user wants to provide a value in a project npmrc to override a user npmrc, with the default? iow, an explicit null